dev-tools/notice/overrides.json

```diff
@@ -1,4 +1,5 @@
 {"name": "github.com/elastic/elastic-agent-client/v7", "licenceType": "Elastic"}
+{"name": "github.com/elastic/elastic-agent-shipper", "licenceType": "Elastic"}
```

Comment:
Oh, this is going to become a potentially circular dependency, since `elastic-agent-shipper` is expected to depend on the beats repository for at least the near-term. It looks like this is only included for the gRPC prototypes, is that right? Can we copy those here directly for now? (I know that's not the cleanest but it's probably ok for this temporary bootstrapping.)